contracts/script/EspToken.s.sol

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import { Script } from "forge-std/Script.sol";
+import { ERC1967Proxy } from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import { EspToken } from "../src/EspTokenV2.sol";
+
+/// @notice Deploys an upgradeable Fee Contract using the OpenZeppelin Upgrades plugin.
+///
+/// To deploy with the ledger, add the --ledger flag to the forge script command:
+///
+/// forge script --ledger DeployEspTokenScript --broadcast
+///
+/// To use a mnemonic
+/// echo test test test test test test test test test test test junk | tr -d '\n' > .mnemonic
+/// forge script --mnemonics .mnemonic DeployEspTokenScript --broadcast
+contract DeployEspTokenScript is Script {
+    string internal contractName = "EspTokenV2.sol";
```

Comment:
i guess this will change pending on the decision from the thread

Reply:
Yeah this is from yesterday. Thanks for catching. I will change it.